cluster/node-pools/master-default/userdata.yaml

```diff
@@ -111,9 +111,7 @@ write_files:
           - --apiserver-count={{ .Values.apiserver_count }}
           - --bind-address=0.0.0.0
           - --insecure-bind-address=0.0.0.0
-{{- if eq .Cluster.ConfigItems.etcd_use_proxy "true" }}
-          - --etcd-servers=http://127.0.0.1:2379
-{{- else if index .NodePool.ConfigItems.etcd_endpoints }}
+{{- if index .NodePool.ConfigItems.etcd_endpoints }}
```

Comment:
I think that's redundant since config items are propagated "down" to the node pools. So if `.Cluster.ConfigItems.etcd_endpoints` is set so will be `.NodePool.ConfigItems.etcd_endpoints`. that would mean we can just use `.NodePool.ConfigItems.etcd_endpoints` here without the if.

Here's an example, global config item with default overwritable in node pool: https://github.com/zalando-incubator/kubernetes-on-aws/search?q=spot_allocation_strategy&unscoped_q=spot_allocation_strategy

Reply:
Good point, fixed.